pkg/controller/keyvault/gateway_test_consts.go

```diff
@@ -0,0 +1,427 @@
+package keyvault
```

Comment:
not positive what the convention for naming a test helper should be but I suspect it should have the _test suffix

Reply:
updated!